src/hooks/useVirtualGrid.js

```diff
@@ -2,10 +2,10 @@ import useVirtualGridDisplay from './useVirtualGridDisplay'
 import useVirtualGridFirstRowIndex from './useVirtualGridFirstRowIndex'
 import useVirtualGridChildren from './useVirtualGridChildren'
 
-const useVirtualGrid = ({ cell, total, viewportRowOffset = 4, onRender }) => {
+const useVirtualGrid = ({ cell, total, viewportRowOffset = 4, onRender, scrollContainer }) => {
   const rowOffset = Math.max(2, Math.round(viewportRowOffset / 2) * 2)
   const { display, style, ref } = useVirtualGridDisplay({ cell, total, rowOffset })
-  const { firstRowIndex, scrolling } = useVirtualGridFirstRowIndex(display)
+  const { firstRowIndex, scrolling } = useVirtualGridFirstRowIndex({ ...display, scrollContainer })
```

Comment:
```suggestion
  const { firstRowIndex, scrolling } = useVirtualGridFirstRowIndex(display, scrollContainer)
```

Reply:
I prefer avoid any unecessary creation of new Object in this library for performances purpose